spec/serializers/api/v1/rejection_reason_serializer_spec.rb

```diff
@@ -6,6 +6,8 @@
   let(:serializer) { Api::V1::RejectionReasonSerializer.new(rejection_reason) }
   let(:json)       { JSON.parse( serializer.to_json ) }
 
+  it_behaves_like 'name_with_language'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.